evalml/pipelines/components/estimators/estimator.py

```diff
@@ -23,6 +24,7 @@ class Estimator(ComponentBase):
     """
     # We can't use the inspect module to dynamically determine this because of issue 1582
     predict_uses_y = False
+    model_family = ModelFamily.NONE
```

Comment:
Wanted some feedback on if this is ok? It subclasses the `model_family` abstract method and is a class property. If it's fine to add this, I'll probably add a custom `component.rst` template

Reply:
Seems like if it doesn't break any tests, it's fine?